src/AppBundle/Controller/Pa/IndexController.php

```diff
@@ -48,6 +49,40 @@ public function dashboardAction(Request $request)
     }
 
     /**
+     * Client edit page
+     * Report is only associated to one client, and it's needed for back link routing,
+     * so it's retrieved with the report with a single API call
+     *
+     * @Route("/client/{clientId}/edit", name="pa_client_edit")
+     * @Template
+     */
+    public function clientEditAction(Request $request, $clientId)
+    {
+        /** @var $client EntityDir\Client */
```

Comment:
in terms of ACL
- route it's under /pa, so Lay deputy cannot access it (see security.yml)
- PAs out of team won't be able to access the client given that ID, see API clientController::findByIdAction, line 98. That exception will be thrown in the controller straightaway